etc/docker/Dockerfile.debian.bullseye

```diff
@@ -10,13 +10,15 @@ RUN apt-get -y dist-upgrade
 RUN apt-get -y --no-install-recommends install \
     build-essential \
     ca-certificates \
-    doxygen \
     curl \
+    doxygen \
```

Comment:
Deltas to the package lists are just sorting and making it more uniform across the four docker images, and adding an explicit dependency on abseil and xtensor when provided by the system.